docs/1-project-setup.md

```diff
@@ -1,25 +1,25 @@
 # 1. Project Setup
 
-This document shows how to create the GitHub repository, Azure resources and configuration of [GitHub Actions](https://help.github.com/en/actions) pipelines necessary to begin developing LUIS models. The setup instructions here are the quickest and the recommended path to getting started.
+This document shows how to create the GitHub repository, Azure resources and configuration of [GitHub Actions](https://help.github.com/en/actions) workflows necessary to begin developing LUIS models. The setup instructions here are the quickest and the recommended path to getting started.
 
 This template is pre-configured with a sample LUIS project ***vacation_requests***, defined in this repo in the [model.lu file](../luis-app/model.lu). The sample LUIS project defines a language understanding model to handle requests for vacation from employees. After running through the setup steps described in this document and the run through of the ["dev inner loop"](https://mitchdenny.com/the-inner-loop/) activities described in [2. Making updates to a LUIS app in a feature branch](2-feature-branches-and-running-pipelines.md), read [3. Customizing the Repository for your own project](3-customizing-own-project.md) for instructions on how to adapt this repository for use with your own solution.
 
 ## Table of Contents
 
 In order to use this template to setup a repo for your own use, you will:
 
-- [Get the code](#get-the-code) - Create your own GitHub repository from this template
+- [Create your repo](#create-your-repo) - Create your own GitHub repository from this template
 - [Clone the repository](#clone-your-repository) to your own machine
 - [Register in the LUIS Authoring portal](#register-in-the-luis-authoring-portal)
 - [Provision Azure resources](#provisioning-azure-resources)
-- [Setup the CI/CD pipeline](#setup-the-ci/cd-pipeline)
-  - [Set environment variables in the pipeline yaml](#set-environment-variables-for-resource-names-in-the-pipeline-yaml)
+- [Setup the CI/CD workflows](#setup-the-cicd-workflows)
+  - [Set environment variables in the workflow yaml](#set-environment-variables-for-resource-names-in-the-workflow-yaml)
```

Comment:
Nit. Do we really need links to these sub topics? Will anyone need to refer directly to them? It makese the process look like it has a lot of steps.